ui.config/src/main/content/META-INF/vault/definition/.content.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
definitions don't need to be part of the package (redundant information, compare with https://github.com/apache/jackrabbit-filevault/blob/master/src/site/markdown/packagedefinition.md)